recipes-kernel/linux/linux-yocto-6.10/0001-NVIDIA-SAUCE-soc-tegra-pmc-Add-sysfs-nodes-to-select.patch

```diff
@@ -0,0 +1,283 @@
+From 36a9d28b3ee946e46d166985ebce16684905f3a5 Mon Sep 17 00:00:00 2001
+From: Petlozu Pravareshwar <petlozup@nvidia.com>
+Date: Fri, 8 Mar 2024 03:18:17 +0000
+Subject: [PATCH] NVIDIA: SAUCE: soc/tegra: pmc: Add sysfs nodes to select boot
+ chain
+
+Add sysfs nodes to select A/B boot chain so that BootROM can select
+the right boot path on warm boot.
+
+Bug 4510385
+
```

Comment:
The yocto-checklayer script doesn't flag an error for this yet, but an `Upstream-Status:`  line is required for all patches.

```
Upstream-Status: Backport [5.15.148-1012.12]
```
inserted here would be the most appropriate. This is more of a forward port than a backport, but "Backport" is the only status allowed by the Yocto tooling that's close enough to describe the situation.
See [here](https://docs.yoctoproject.org/singleindex.html#patch-upstream-status) for more info, and note that the tools are annoyingly case-sensitive in their checks.

Reply:
Understood ! Thank you for the feedback I have made the changes.